frontend/demo/src/actions/messages/index.ts

```diff
@@ -38,6 +44,24 @@ export function listMessages(conversationId: string) {
   };
 }
 
+export function sendMessages(messagePayload: SendMessagesRequestPayload) {
+  return async (dispatch: Dispatch<any>) => {
+    return HttpClientInstance.sendMessages(messagePayload)
+      .then((response: Message) => {
+        dispatch(
+          sendMessagesAction({
+            conversationId: messagePayload.conversationId,
+            message: response,
+          })
+        );
+        return Promise.resolve(true);
+      })
+      .catch((error: Error) => {
+        return Promise.reject(error);
```

Comment:
this is unneeded and the catch can be gone. The promise does this reject thing automatically